src/index.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route
+} from "react-router-dom";
 
 import './stylesheets/application.scss';
 
-import HeaderImg from "./static/images/header.jpeg";
+import Header from './components/Header';
+import NotFound from './pages/NotFound';
 
 const App = () => {
   return (
-    <div className="app">
-      <img alt="header" src={HeaderImg} className="app-header" />
-      <p>
-        We are a most promising species, Mr. Spock, as predators go. Did you know that? I frequently
-        have my doubts. I dont. Not any more. And maybe in a thousand years or so, we will be able
-        to prove it.
-      </p>
-      <p>- Captain Kirk</p>
-    </div>
+    <Router>
+      <Header />
+
+      <div className="main-body">
+        <Switch>
+          <Route exact path="/">
+            <Home />
```

Comment:
If you don't pass any props to this component, you can use simplified version `<Route exact path="/" component={Home}`/>

https://reacttraining.com/react-router/web/api/Route/route-render-methods

Reply:
Done